viewer/src/App.tsx

```diff
@@ -21,6 +21,7 @@ const samples = [
     { id: "basic-component", title: "Basic Component" },
     { id: "basic-service", title: "Basic Service" },
     { id: "commands-and-operations", title: "Commands and Operations" },
+    { id: "previous-viewpoint", title: "Go to Previous Viewpoint" },
```

Comment:
It might be more helpful to name the sample around the concept(s) that it illustrates, rather than what it happens to do.

Reply:
Updated.